### PR TITLE
Move ApiDomain to global section of the config (Issue #36)

### PR DESCRIPTION
**Pull request by myles-mcleroy**
This commit moves the API domain from `local.settings.json` to `Settings.json`. To do so, I added two methods to the ConfigManager (GetApiDomainToken() and GetApiDomain()). In the setup of ConfigManager I created a map from the DefaultKey to the ApiDomainToken. I also increased coverage in ConfigManagerTests and GithubConfigManagerTests.

Since this is my first change to the repository, I wasn't 100% confident on where I should implement these changes (ConfigManager or GithubConfigManager), or the best way to do so (such as using a map instead of a plain string, or using try/catch to handle returning the apiDomain as a string). I'm open to suggestions on improving my code and more than happy to move things around.

### CODE REVIEW

**Comment by myles-mcleroy** [comment]
I don't see my changes to `ConfigManager`. Let me update the commit.

**Comment by myles-mcleroy** [comment]
My changes to `ConfigManager` and `ConfigManagerTests` are in `CEDAR.Core.Collector` under [this](https://github.com/microsoft/CEDAR.Core.Collector/pull/35) pull request.

File: GitHub.Collectors.Functions/Settings.template.json [hunk 1]
```diff
@@ -1,33 +0,0 @@
-{
```
Comment (kivancmuslu) [inline GitHub.Collectors.Functions/Settings.template.json:1]: Wondering, why did you delete this file?

File: GitHub.Collectors.Functions/Settings.barebones.template.json [hunk 2]
```diff
@@ -22,5 +22,6 @@
         "TrafficTimer": {},
 
         "Traffic": {}
-    }
+    },
+    "ApiDomain":  "api.github.com"
```
Comment (kivancmuslu) [inline GitHub.Collectors.Functions/Settings.barebones.template.json:26]: We need this to be added to all Ring configurations as well, right (to the actual production deployment)?

**Review by kivancmuslu — CHANGES_REQUESTED** [review]
I think you need to also update all Ring configurations to have proper API domain values, could you please check?

File: GitHub.Collectors.Functions/Settings.barebones.template.json (hunk 2 above)
Comment (kivancmuslu) [reply GitHub.Collectors.Functions/Settings.barebones.template.json:26]: I believe this is resolved, right (in the ADO side)?

**Review by kivancmuslu — APPROVED** [review]
:shipit:

**Pull request merged**